turn.go

```diff
@@ -212,7 +212,11 @@ func (s *Server) handleAllocateRequest(conn net.PacketConn, srcAddr net.Addr, m
 	// Check current usage vs redis usage of other servers
 	// if bad, redirect { stun.AttrErrorCode, 300 }
 	lifetimeDuration := allocationLifeTime(m)
-	a, err := s.manager.CreateAllocation(fiveTuple, conn, requestedPort, lifetimeDuration)
+	a, err := s.manager.CreateAllocation(
+		fiveTuple, conn,
+		s.relayIPs[0], // TODO: allow more than one relay IP
```

Comment:
I think `conn.LocalAddr`'s IP is ok here. But I have another question, if the turn server have multiple public `IP`， how to get the correspondence between public IP and intranet IP.

Reply:
Applications like TURN server or "routers" may have the bandwidth bottleneck at network interfaces (NIC, Ethernet, etc), and it is pretty common to have separate interfaces (just like routers, switches have multiple ports). So, I think it is a good idea to allow having TURN (listening) ports and relay ports.

CoTurn for instance provides that option also. See [turnserver.conf](https://github.com/coturn/coturn/blob/master/examples/etc/turnserver.conf)

Reply:
Cool, I see the configuration `external-ip=60.70.80.92/172.17.19.102` can solve this problem.

Reply:
For that, I filed an issue as #56.
These are the ones I was talking about for separate relay IP options:
```
#relay-ip=172.17.19.105
#relay-ip=2607:f0d0:1002:51::5
```
FYI